build/LegacyPackages/14.0/Microsoft.Build.Conversion.Core/Microsoft.Build.Conversion.Core.nuspec

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata minClientVersion="2.8.1">
+    <id>$id$</id>
+    <version>$version$</version>
+    <title>$id$</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>
+    <projectUrl>http://go.microsoft.com/fwlink/?LinkId=624683</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <description>This package contains the $id$ assembly which contains logic for converting projects.</description>
```

Comment:
Mention obsolescence? Looks like @tmeschter added `NOTE: this package is deprecated.` in [https://github.com/Microsoft/msbuild/pull/904/files#diff-9da9f874a90e5a380697d45d83f162f4R15](https://github.com/Microsoft/msbuild/pull/904/files#diff-9da9f874a90e5a380697d45d83f162f4R15)